usaspending_api/awards/models.py

```diff
@@ -52,10 +53,8 @@ class FinancialAccountsByAwards(DataSourceTrackedModel):
     treasury_account = models.ForeignKey(TreasuryAppropriationAccount, models.CASCADE, null=True)
     submission = models.ForeignKey(SubmissionAttributes, models.CASCADE)
     award = models.ForeignKey('awards.Award', models.CASCADE, null=True, related_name="financial_set")
-    program_activity_name = models.CharField(max_length=164, blank=True, null=True)
-    program_activity_code = models.ForeignKey(RefProgramActivity, models.DO_NOTHING, db_column='program_activity_code', blank=True, null=True)
-    object_class = models.ForeignKey(RefObjectClassCode, models.DO_NOTHING, null=True, db_column='object_class')
-    by_direct_reimbursable_funding_source = models.CharField(max_length=1, blank=True, null=True)
```

Comment:
As far as I can tell this wasn't moved to anywhere else - was it simply dropped for being unused / unimportant?